### ISSUE TITLE
Doctype/html

### ISSUE THREAD

**Issue by tenbits** [mwilliamson/mammoth.js#52]
Hi Michael, would be it possible to wrap the result to the real HTML document, with `doctype`, `html`, `body`? Also it would be nice to have `<meta charset="UTF-8">` in `head` element.
Thanks

**Comment by mwilliamson** [comment]
It's possible, but I'm not sure what value it would add beyond allowing the user to do so themselves, given that each user's situation is likely to be quite different. Is there a particular reason you'd want Mammoth to handle the wrapping, rather than handling it yourself?

**Comment by tenbits** [comment]
I would suggest to handle this via the configuration flag, for instance `--document`. We use `mammoth` in a bash script to convert a lot of docx to html. That's why we had to add to the pipeline, a script to wrap all the files with proper `html` document declaration. With `charset utf8`.
And it would be very nice, if the feature would be out of the box.

**Comment by mwilliamson** [comment]
It seems to me that the appropriate place for this is in your own bash script since that's the point at which you know where the HTML is going to be used. Presumably you have something like:

```
echo '<html><body>' >> $OUTPUT_FILE
mammoth path/to/document >> $OUTPUT_FILE
echo '</body></html>' >> $OUTPUT_FILE
```

which seems relatively straightforward, and simpler than the changes that would be required to Mammoth. Having said that, I'd consider a pull request if you still thought it was worth it and were willing to add it.

**Comment by tenbits** [comment]
You are right, that it is easy to find a solution for this. But such a feature should belong also to the library. As converting word document to html, it is not obvious not getting a document (_html_).
Anyway, great job, thanks for doing this!

**Comment by cymen** [comment]
I'd really like this too -- I did a conversion today, opened the result in a browser and was confused why UTF-8 was getting munged. Then I realized after about 30-45 minutes (partly while looking at other things), that it was actually fine in the output file and displaying wrong in the browser due to a lack of a doctype setting encoding to UTF-8.

**Comment by cymen** [comment]
After reading the thread, at least a warning in the README would be sensible -- first thing I did was come to here and check the README. I'll do a PR for that.
